client/app/scripts/superdesk-archive/directives.js

```diff
@@ -243,6 +243,15 @@
                             familyService.fetchDesks(scope.item, false)
                                 .then(function(desks) {
                                     scope.desks = desks;
+                                    scope.tooltipText = '';
+
+                                    for (var i = 1; i < desks.length; i++) {
```

Comment:
It's on purpose, first desk should be shown as link, and other desks as tooltip.